### DISCUSSION TITLE
Aggrigate multiple alerts from same soure and rule with elastalert

### DISCUSSION THREAD

**Discussion by sleepingbel** [Security-Onion-Solutions/securityonion#8266] [Unsupported Versions] [answered]
Hello all,

Is it possible to aggrigate alerts from the same rule (in my case sigma rule) and from the same source in elastalert? if needed in a specific timeframe example 10 min

If yes where can i find the documentation?

Regards
Bart

**Comment by btdewey** [comment, marked as answer]
I'm not certain what you mean by aggregate.  Are you only wanting the rule to fire if there are a certain number of events or only fire once every 10 minutes if it's from the same source?

**Reply by sleepingbel** [reply]
only fire once every 10 min when it happens

**Reply by btdewey** [reply]
This might be better asked in the SigmaHQ discussions here, but the way I'm reading the specification there isn't a function in Sigma to do what you want.  I don't think the sigma correlations is what you want, but I linked it below just in case.
https://github.com/SigmaHQ/sigma/discussions
 https://github.com/SigmaHQ/sigma/wiki/Specification:-Sigma-Correlations

The elastalert aggregations seems to be what your looking for, but I haven't manually configured elastalerts on SO.  The playbooks use sigma to create the elastalerts and I haven't been able to successfully manually modify the elastalert once it's generated from the sigma code.
https://elastalert.readthedocs.io/en/latest/ruletypes.html

**Reply by sleepingbel** [reply]
Thx for the info
